common/src/main/java/com/wynntils/features/user/overlays/CustomBarsOverlayFeature.java

```diff
@@ -139,6 +146,25 @@ protected void renderBar(PoseStack poseStack, float renderY, float progress) {
                     progress);
         }
 
+        public void renderColoredBar(PoseStack poseStack, float renderY, CustomColor color, float progress) {
```

Comment:
What is the reasoning for introducing `renderColoredBar` when `renderBar` already exists?

Reply:
It renders a bar which can be colored. There is no point in having it in the HealthBarOverlay class though. See my comments.

Reply:
`renderBar` runs` RenderUtils.drawProgressBar`, whilst `renderColoredBar` runs `RenderUtils.drawColoredProgressBar`